src/olympia/lib/settings_base.py

```diff
@@ -541,15 +541,27 @@ def get_db_config(environ_var, atomic_requests=True):
             # it's not in templates/ with a .html extension.
             (EDITORIAL_CONTENT_FILENAME, 'jinja2'),
 
-            # Make sure we're parsing django-admin templates with the django
-            # template extractor
+            # Make sure we're parsing django-admin & email templates with the
+            # django template extractor. This should match the behavior of
+            # JINJA_EXCLUDE_TEMPLATE_PATHS
+            (
+                'src/olympia/**/templates/**/emails/**.*',
+                'django_babel.extract.extract_django'
+            ),
             (
                 '**/templates/admin/**.html',
                 'django_babel.extract.extract_django'
             ),
+            (
+                '**/templates/addons/admin/**.html',
+                'django_babel.extract.extract_django'
+            ),
+            (
+                '**/templates/blocklist/**.html',
+                'django_babel.extract.extract_django'
+            ),
 
             ('src/olympia/**/templates/**.html', 'jinja2'),
-            ('**/templates/**.lhtml', 'jinja2'),
```

Comment:
Why was this one removed?

Reply:
This only matched email templates, which are now processed by django template language instead of jinja2 (https://github.com/mozilla/addons-server/pull/14045)